workflow/Snakefile

```diff
@@ -505,11 +533,69 @@ rule multiqc:
         -fz {raw_dir} {trim_dir} {align_dir} {quant_dir} {processed_dir}
         mv MultiQC_Report.html MultiQC_Report_data.zip {qc_dir}
         '''
+        
+#####################
+### Study-wide QC ###
+#####################
+
+rule studywideqc:
+    message: "generating study-wide QC report in R..."
+    input:
+        qc_dir / "MultiQC_Report.html",
+        processed_dir / "count_table.tsv",
+        "genome.removed",
+    output:
+        qc_dir / "details/samples_removed.txt"
+    conda:
+        "envs/reports.yml"
+    benchmark: log_dir / "benchmark.studywide_qc.txt"
+    shell:
+        '''
+        Rscript scripts/render_studywide_QC_report.R
+        '''
+
+##############
+### DESeq2 ###
+##############
+
+rule deseq2:
+    message: "running DESeq2..."
+    input:
+        qc_dir / "details/samples_removed.txt"
+    output:
+        touch("DESeq2_complete")
+    conda:
+        "envs/reports.yml"
+    benchmark: log_dir / "benchmark.deseq2.txt"
+    shell:
+        '''
+        Rscript scripts/run_DESeq2.R 
+        '''
+        
+#####################
+### Study-wide QC ###
+#####################
```

Comment:
fix name here

Reply:
I know you said there was cleanup to do but my brain wouldn't let me not note this 😅